test/extended/router/scoped.go

```diff
@@ -460,3 +514,73 @@ func createOverrideRouterPod(routerImage string) *corev1.Pod {
 		},
 	}
 }
+
+func createOverrideDomainRouterPod(routerImage string) *corev1.Pod {
+	return &corev1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "Pod",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "router-override-domains",
+			Labels: map[string]string{
+				"test": "router-override-domains",
+			},
+		},
+		Spec: corev1.PodSpec{
+			TerminationGracePeriodSeconds: utilpointer.Int64(1),
+			Containers: []corev1.Container{
+				{
+					Name:            "route",
```

Comment:
I don't think it breaks anything, but you changed the container name from "router" in `router-override-domains.yaml` to "route" in this Go definition.

Reply:
Fixed